src/config/mod.rs

```diff
@@ -160,32 +171,30 @@ pub async fn create_client_builder(options: ConfigOptions) -> Result<(ClientBuil
     }
 
     Ok((client_builder.default_headers(headers), loader))
-
 }
 
 /// Returns a config which is used by clients within pods on kubernetes.
 ///
 /// It will return an error if called from out of kubernetes cluster.
 pub fn incluster_config() -> Result<Configuration> {
-    let server = incluster_config::kube_server().ok_or_else(||
+    let server = incluster_config::kube_server().ok_or_else(|| {
         Error::KubeConfig(format!(
             "Unable to load incluster config, {} and {} must be defined",
             incluster_config::SERVICE_HOSTENV,
             incluster_config::SERVICE_PORTENV
-    )))?;
+        ))
+    })?;
 
-    let ca = incluster_config::load_cert()
-        .map_err(|e| Error::SslError(format!("{}", e)))?;
+    let ca = incluster_config::load_cert().map_err(|e| Error::SslError(format!("{}", e)))?;
     let der = ca.to_der().map_err(|e| Error::SslError(format!("{}", e)))?;
-    let req_ca = Certificate::from_der(&der)
-        .map_err(|e| Error::SslError(format!("{}", e)))?;
+    let req_ca = Certificate::from_der(&der).map_err(|e| Error::SslError(format!("{}", e)))?;
 
     let token = incluster_config::load_token()
         .map_err(|e| Error::KubeConfig(format!("Unable to load in cluster token: {}", e)))?;
 
-    let default_ns = incluster_config::load_default_ns().map_err(|e| Error::KubeConfig(
-        format!("Unable to load incluster default namespace: {}", e),
-    ))?;
+    let default_ns = incluster_config::load_default_ns().map_err(|e| {
+        Error::KubeConfig(format!("Unable to load incluster default namespace: {}", e))
+    })?;
```

Comment:
Though all of this would have to have merge conflicts resolved in the rustls branch. Do you have to use rustfmt on the whole file?

Reply:
No I don't, my editor just does this automatically and I appreciate it with my own work. I will amend the commit to only the true affected lines.

Reply:
Yeah, I like that automatic setup as well. Will want to get the project properly formatted at some point so that we can use this in that way.

..but while we have outstanding big branches, let's hold off.